### ISSUE TITLE
why my model is not showing in the pytorch website 

### ISSUE THREAD

**Issue by tahmeedtoqi** [pytorch/hub#346]
i have PR my "Add ToqiNet Model to PyTorch Hub #345" is successfully merged as it shown but regardlesss from the docs of hub it should be able to show "We accept submission to PyTorch hub through PR in hub repo. Once the PR is merged into master here, it will show up on the [PyTorch website](https://pytorch.org/hub) in 24 hrs.". so its been a 24 hrs and i still didnt find my model in pytorch website.

Thank you

**Comment by NicolasHug** [comment]
Hi @tahmeedtoqi , the reason the model doesn't show on the website is because https://github.com/pytorch/hub/pull/345 wasn't merged. Since the corresponding PR was closed, I'll close this issue as well.
